app/controllers/user_sessions_controller.rb

```diff
@@ -4,11 +4,12 @@ def new
   end
 
   def create
-    @user = User.find_by(firstname: params[:user][:firstname])
+    @user = User.find_by(first_name: params[:user][:first_name])
```

Comment:
instead of first_name, isn't it better to do it with the email? Because first name can be same for many people?